pkgs/by-name/uu/uuu/package.nix

```diff
@@ -30,7 +30,7 @@ stdenv.mkDerivation (finalAttrs: {
   passthru.updateScript = nix-update-script {
     extraArgs = [
       "--version-regex"
-      "uuu_\\([0-9.]+\\)"
+      "uuu_\([0-9.]+\)"
```

Comment:
Either you want a backslash, in which case `\\(` is correct, or you don't, then write `(` directly.

```
nix-repl> "uuu_\([0-9.]+\)"
"uuu_([0-9.]+)"
```